docs/dev-guides/7-xrpackage-api.md

```diff
@@ -53,6 +60,13 @@ The `XRPackageEngine` API is documented [in the next section](./8-xrpackage-engi
 
 ## `getVolumeMesh`
 
+Retrieve the collision mesh of the XRPackage if it exists.
+
+**Parameters**: None
+
+**Returns**: A <a href="https://threejs.org/docs/#api/en/scenes/Scene" target="_blank" rel="noopener noreferrer">`Scene`</a> object in JSON format with metadata for the package if a volume mesh exists, or `null` if a volume mesh does not exist. See <a href="https://github.com/mrdoob/three.js/wiki/JSON-Object-Scene-format-4" target="_blank" rel="noopener noreferrer">`JSON Object Scene Format`</a> and <a href="https://github.com/mrdoob/three.js/blob/master/src/scenes/Scene.js" target="_blank" rel="noopener noreferrer">`Scene.js`</a> for more information.
```

Comment:
Is this actually that scene format? I don't think it is. I think it's probably the actual mesh, even though it might be converted to that scene format when you JSONify it, by using a JSON trap handler.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
Oh I see what you mean, would it be better to just specify it as a Scene object then and not some JSON?

Reply:
Yup, looks good now. Thanks!